core/src/main/java/de/danoeh/antennapod/core/util/gui/NotificationUtils.java

```diff
@@ -23,113 +24,115 @@
-        notificationChannel.setDescription(c.getString(R.string.notification_channel_playing_description));
-        notificationChannel.setShowBadge(false);
-        return notificationChannel;
+    private static NotificationChannelCompat createChannelPlaying(final Context c) {
+        return new NotificationChannelCompat.Builder(
+                CHANNEL_ID_PLAYING, NotificationManagerCompat.IMPORTANCE_LOW)
+                .setName(c.getString(R.string.notification_channel_playing))
+                .setDescription(c.getString(R.string.notification_channel_playing_description))
+                .setShowBadge(false)
+                .build();
     }
 
-    @RequiresApi(api = Build.VERSION_CODES.O)
-    private static NotificationChannel createChannelError(Context c) {
-        NotificationChannel notificationChannel = new NotificationChannel(CHANNEL_ID_DOWNLOAD_ERROR,
-                c.getString(R.string.notification_channel_download_error), NotificationManager.IMPORTANCE_HIGH);
-        notificationChannel.setDescription(c.getString(R.string.notification_channel_download_error_description));
-        notificationChannel.setGroup(GROUP_ID_ERRORS);
+    private static NotificationChannelCompat createChannelError(final Context c) {
+        final NotificationChannelCompat.Builder notificationChannel
+                = new NotificationChannelCompat.Builder(
+                CHANNEL_ID_DOWNLOAD_ERROR, NotificationManagerCompat.IMPORTANCE_HIGH)
+                .setName(c.getString(R.string.notification_channel_download_error))
+                .setDescription(c.getString(R.string.notification_channel_download_error_description))
+                .setGroup(GROUP_ID_ERRORS);
 
         if (!UserPreferences.getShowDownloadReportRaw()) {
             // Migration from app managed setting: disable notification
-            notificationChannel.setImportance(NotificationManager.IMPORTANCE_NONE);
+            notificationChannel.setImportance(NotificationManagerCompat.IMPORTANCE_NONE);
         }
-        return notificationChannel;
+        return notificationChannel.build();
     }
 
-    @RequiresApi(api = Build.VERSION_CODES.O)
-    private static NotificationChannel createChannelSyncError(Context c) {
-        NotificationChannel notificationChannel = new NotificationChannel(CHANNEL_ID_SYNC_ERROR,
-                c.getString(R.string.notification_channel_sync_error), NotificationManager.IMPORTANCE_HIGH);
-        notificationChannel.setDescription(c.getString(R.string.notification_channel_sync_error_description));
-        notificationChannel.setGroup(GROUP_ID_ERRORS);
+    private static NotificationChannelCompat createChannelSyncError(final Context c) {
+        final NotificationChannelCompat.Builder notificationChannel
+                = new NotificationChannelCompat.Builder(
+                CHANNEL_ID_SYNC_ERROR, NotificationManagerCompat.IMPORTANCE_HIGH)
+                .setName(c.getString(R.string.notification_channel_sync_error))
```

Comment:
Please do not put the assignment into a new line. Also, aligning the parameters of a function with stuff that happens after the function is called could be misleading. So please indent that a bit. Same for the other channels/groups in this file.

```suggestion
        final NotificationChannelCompat.Builder notificationChannel = new NotificationChannelCompat.Builder(
                    CHANNEL_ID_SYNC_ERROR, NotificationManagerCompat.IMPORTANCE_HIGH)
                .setName(c.getString(R.string.notification_channel_sync_error))
```

Reply:
Sure.